tutorials/algorithms/02_vqe_advanced_options.ipynb

```diff
@@ -0,0 +1,541 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Callback"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "Callback methods can be used to monitor optimization progress as the algorithm runs and converges to the minimum. The callback is invoked for each functional evaluation by the optimizer and provides the current optimizer value, evaluation count, current optimizer parameters etc. Note that, depending on the specific optimizer this may not be each iteration (step) of the optimizer, so for example if the optimizer is calling the cost function to compute a finite difference based gradient this will be visible via the callback.\n",
+    "\n",
+    "This section demonstrates how to leverage callbacks in `VQE` to plot the convergence path to the ground state energy with a selected set of optimizers."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we create a qubit operator for VQE. Here we will use the same operator as used in the algorithms introduction, which was originally computed by Qiskit Nature for an H2 molecule."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit.quantum_info import SparsePauliOp\n",
+    "\n",
+    "H2_op = SparsePauliOp.from_list(\n",
+    "    [\n",
+    "        (\"II\", -1.052373245772859),\n",
+    "        (\"IZ\", 0.39793742484318045),\n",
+    "        (\"ZI\", -0.39793742484318045),\n",
+    "        (\"ZZ\", -0.01128010425623538),\n",
+    "        (\"XX\", 0.18093119978423156),\n",
+    "    ]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We then instantiate our `Estimator` of choice for the evaluation of expectation values within `VQE`. In this case, we will use the `qiskit.primitives.Estimator` shipped with the default Qiskit Terra installation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "from qiskit.primitives import Estimator\n",
+    "\n",
+    "estimator = Estimator()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will now see the VQE callback usage over a set of optimizers for comparison. The minimum energy of the H2 Hamiltonian can be found quite easily, so the value of maxiters does not have to be very large. We will once again use `TwoLocal` as our selected trial wavefunction (i.e. ansatz)."
```

Comment:
```suggestion
    "We will now see the VQE callback usage over a set of optimizers for comparison. The minimum energy of the H2 Hamiltonian can be found quite easily, so the value of `maxiters` does not have to be very large. We will once again use `TwoLocal` as our selected trial wavefunction (i.e. ansatz)."
```

Reply:
The variable is `maxiter` so if we want to have it highlighted I guess it should be correct

Reply:
Fixed